phpstan.neon

```diff
@@ -3,7 +3,7 @@ includes:
     - tools/vendor/phpstan/phpstan-phpunit/rules.neon
 
 parameters:
-    phpVersion: 80100
+    phpVersion: 80200
```

Comment:
You can remove this line because PHPStan will follow this too: https://github.com/Roave/BetterReflection/blob/7f16c808bb64f34b172d14966e348ccdcd97f8b8/composer.json#L64